plugins/outputs/bigquery/bigquery.go

```diff
@@ -89,13 +89,11 @@ func (s *BigQuery) setUpDefaultClient() error {
 	var credentialsOption option.ClientOption
 
 	ctx := context.Background()
-	ctx, cancel := context.WithTimeout(ctx, time.Duration(s.Timeout))
-	defer cancel()
```

Comment:
Why do you remove the timeout?

Reply:
From the cloud [docs](https://cloud.google.com/go/docs/reference/cloud.google.com/go/0.94.1#hdr-Timeouts_and_Cancellation), 
```
Do not attempt to control the initial connection (dialing) of a service by setting a timeout on the context passed to NewClient. Dialing is non-blocking, so timeouts would be ineffective and would only interfere with credential refreshing, which uses the same context.
```

Reply:
Done, not sure why the checks didn't trigger though.

Reply:
Please run `make fmt`, seems like go has some different view on how things should be formatted. ;-)

Reply:
Done